build-system/pr-check/remote-tests.js

```diff
@@ -29,6 +29,7 @@ const {
   stopTimer,
   startSauceConnect,
   stopSauceConnect,
+  timedExec,
```

Comment:
Minor nit for future cleanup: `timedExec` prints the wrong filename in the logs. Might need something like:

https://github.com/ampproject/amphtml/blob/f0de64b1bf8729b18e778c0801535bdc41610fa0/build-system/pr-check/remote-tests.js#L40-L41